src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -51,7 +52,14 @@ internal EnhancedImageServer() : base( new RuntimeImageProcessor(new BookRenamed
 		{ }
 
 		public EnhancedImageServer(RuntimeImageProcessor cache): base(cache)
+		{ }
+
+		/// <summary>
+		/// This constructor is used for unit testing
+		/// </summary>
+		public EnhancedImageServer(RuntimeImageProcessor cache, BloomFileLocator fileLocator) : base(cache)
```

Comment:
If it's only used and intended for unit testing, we can make it internal, I believe. There is a special trick, which I think is turned on in Bloom.exe, to allow the test DLL to use Bloom's internals. Not worth a new round of changes, but for future reference and in case something else needs fixing...